architecture-building-systems/CityEnergyAnalyst
# I2021 pumping energy

the pumping energy calculation has been updated, only the pressure losses in the critical path is being considered now.
to test: (please test one DH and one DC case)
1. run `cea thermal-network` 
2. check all the thermal network plots from the interface
